examples/website/src/shared/blocks/richTextBlock/editor/render.tsx

```diff
@@ -53,7 +53,12 @@ export function renderElement({attributes, children, element}: RenderElementProp
       // title={title}
 
       return (
-        <a data-title={element.title} data-href={element.url} {...attributes}>
+        <a
+          target="_blank"
+          rel="noreferrer"
+          href={element.url as string}
```

Comment:
Is that change because of the change in the npm module we use?

Reply:
it's because Pascal replaced the original richText render block with the richTextEditor block from the edtior package